tests/datatypes/test_bytearray.py

```diff
@@ -242,248 +115,53 @@ class BinaryBytearrayOperationTests(BinaryOperationTestCase, TranspileTestCase):
         'test_subscr_slice',
         'test_subscr_str',
         'test_subscr_tuple',
-
-        'test_subtract_bool',
-        'test_subtract_bytearray',
-        'test_subtract_bytes',
-        'test_subtract_class',
-        'test_subtract_complex',
-        'test_subtract_dict',
-        'test_subtract_float',
-        'test_subtract_frozenset',
-        'test_subtract_int',
-        'test_subtract_list',
-        'test_subtract_None',
-        'test_subtract_NotImplemented',
-        'test_subtract_range',
-        'test_subtract_set',
-        'test_subtract_slice',
-        'test_subtract_str',
-        'test_subtract_tuple',
-
-        'test_true_divide_bool',
-        'test_true_divide_bytearray',
-        'test_true_divide_bytes',
-        'test_true_divide_class',
-        'test_true_divide_complex',
-        'test_true_divide_dict',
-        'test_true_divide_float',
-        'test_true_divide_frozenset',
-        'test_true_divide_int',
-        'test_true_divide_list',
-        'test_true_divide_None',
-        'test_true_divide_NotImplemented',
-        'test_true_divide_range',
-        'test_true_divide_set',
-        'test_true_divide_slice',
-        'test_true_divide_str',
-        'test_true_divide_tuple',
-
-        'test_xor_bool',
-        'test_xor_bytearray',
-        'test_xor_bytes',
-        'test_xor_class',
-        'test_xor_complex',
-        'test_xor_dict',
-        'test_xor_float',
-        'test_xor_frozenset',
-        'test_xor_int',
-        'test_xor_list',
-        'test_xor_None',
-        'test_xor_NotImplemented',
-        'test_xor_range',
-        'test_xor_set',
-        'test_xor_slice',
-        'test_xor_str',
-        'test_xor_tuple',
     ]
 
+    not_implemented_versions = {
+        'test_modulo_bool': ['3.5', '3.6'],
+        'test_modulo_bytearray': ['3.5', '3.6'],
+        'test_modulo_bytes': ['3.5', '3.6'],
+        'test_modulo_class': ['3.5', '3.6'],
+        'test_modulo_complex': ['3.5', '3.6'],
+        'test_modulo_dict': ['3.5', '3.6'],
+        'test_modulo_float': ['3.5', '3.6'],
+        'test_modulo_frozenset': ['3.5', '3.6'],
+        'test_modulo_int': ['3.5', '3.6'],
+        'test_modulo_list': ['3.5', '3.6'],
+        'test_modulo_None': ['3.5', '3.6'],
+        'test_modulo_NotImplemented': ['3.5', '3.6'],
+        'test_modulo_range': ['3.5', '3.6'],
+        'test_modulo_set': ['3.5', '3.6'],
+        'test_modulo_slice': ['3.5', '3.6'],
+        'test_modulo_str': ['3.5', '3.6'],
+        'test_modulo_tuple': ['3.5', '3.6'],
+    }
```

Comment:
I'm guessing this is the result of 3.5 introducing string interpolation for bytearray?

Reply:
Yeah. These are the only behaviours on Bytearray that aren't in place yet.